molgenis-util/src/test/java/org/molgenis/util/mail/MailSenderImplTest.java

```diff
@@ -48,4 +48,11 @@ public void testSendTwoMessages()
 		mailSender.send(simpleMailMessage, secondSimpleMailMessage);
 		verify(actualMailSender).send(simpleMailMessage, secondSimpleMailMessage);
 	}
+
+	@Test(expectedExceptions = MailException.class)
+	public void failAtMailSenderConstruction()
+	{
+		doThrow(new IllegalStateException("fail!")).when(mailSenderFactory).createMailSender(mailSettings);
+		mailSender.send(simpleMailMessage);
```

Comment:
i dont think i understand , what is the use of this test , i looks like its testing a mocked call